src/Microsoft.AspNetCore.Razor.Runtime/Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -711,7 +711,7 @@ private static bool IsAccessibleProperty(PropertyInfo property)
         /// ONE1TWO2THREE3 => one1two2three3
         /// First_Second_ThirdHi => first_second_third-hi
         /// </example>
```

Comment:
Need to fix the `<example>` since this is now `public`. Right now all the lines will be shown as one messy line. Easiest to surround the lines w/ `<code>...</code>`.